sos/report/plugins/hyperv.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Hyperv(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Hyper-V client information"""
+
+    plugin_name = "hyperv"
+    files = ('/sys/bus/vmbus/', '/boot/config-*')
```

Comment:
What about enabling the plugin on presence of the `hyperv-tools` package? That appears to be what provides `lsvmbus` (at least on Fedora, would need to verify package names on the other distros). This would also allow for package verification when sos is run with `--verify`.

Reply:
Following on this - are there any other commands that ship with the `hyperv-tools` package that should be collected, beyond `lsvmbus`?

Reply:
Bizarrely the package seems to include exactly one file, the `lsvmbus` binary:

```
$ rpm -ql hyperv-tools
/usr/sbin/lsvmbus
```

Reply:
Ah, ok - it's a sub-package of the `hyperv-daemons` SRPM.

Reply:
On Ubuntu I believe `lsvmbus` is included in the `linux-cloud-tools-generic` package. I would like the plugin to collect the `/sys/bus/vmbus` files even if `lsvmbus` is unavailable. Also there are not very many cases of a Hyper-V guest not having the `/sys/bus/vmbus` directory, so the case of a machine having `lsvmbus` and not `/sys/bus/vmbus` may not be important to consider.

Reply:
Jake, to your question about other tools in `hyperv-tools` (and `linux-cloud-tools-generic` etc.), `lsvmbus` is the only one which provides useful information.